public/components/marketing-cards/29.html

```diff
@@ -0,0 +1,17 @@
+<div
+  class="flex w-full aspect-[3/2] flex-col gap-4 overflow-hidden rounded-lg bg-gray-100 shadow-md transition-all hover:scale-[101%] hover:shadow-lg"
+>
+  <div class="h-1/2">
+    <div class="relative h-2/3 bg-[#9B1C31]">
+      <img
+        src="https://images.pexels.com/photos/1998865/pexels-photo-1998865.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+        alt=""
+        class="absolute left-[50%] bottom-0 aspect-[1/1] w-[30%] -translate-x-1/2 translate-y-1/2 rounded-full border-4 border-gray-100 object-cover shadow-md"
+      />
+    </div>
+  </div>
+  <div class="flex h-1/2 flex-col items-center justify-center">
```

Comment:
Why the use of `flex` classes and then `text-center`? Just the use of `text-center` would do the job.